ui/packages/consul-nspaces/app/templates/dc/nspaces/index.hbs

```diff
@@ -64,8 +64,8 @@ as |route|>
       <BlockSlot @name="toolbar">
       {{#if (gt items.length 0)}}
         <Consul::Nspace::SearchBar
-          @search={{search}}
-          @onsearch={{action (mut search) value="target.value"}}
+          @search={{this.search}}
+          @onsearch={{action (mut this.search) value="target.value"}}
```

Comment:
[note]: ideally we would address two items in the follow up as well, since it would help with the next steps:
* remove all `action` helpers
* remove all `mut` helpers